packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -1137,7 +1137,7 @@ Adds an OAuth 2.0 client application. This application is only available to the
 | grant_types                                 | Array of OAuth 2.0 grant type strings                                                                                                                                                                                      | Array of `authorization_code`, `implicit`, `password`, `refresh_token`, `client_credentials`   | FALSE      | FALSE    | TRUE       |
 | initiate_login_uri                          | URL string that a third party can use to initiate a sign in by the client                                                                                                                                                    | String                                                                                         | TRUE       | FALSE    | TRUE       |
 | issuer_mode <ApiLifecycle access="ea" />    | Indicates whether the Okta Authorization Server uses the original Okta org domain URL or a custom domain URL as the issuer of ID token for this client. See [Details](#details). | `CUSTOM_URL` or `ORG_URL`                                                                      | TRUE       | FALSE    | TRUE       |
-| logo_uri                                    | URL string that references a logo for the client. This value is automatically copied from any logo you provide in the App Wizard for **Application logo**, but can be changed to a different URI for consent.               | URL                                                                                            | TRUE       | FALSE    | FALSE      |
+| logo_uri                                    | URL string that references a logo for the client.<br><br> This value is used with the client consent dialog box during the client consent flow when you:<ul><li> Create an app using the API and specify a value for `logo_uri`</li><li>Create an app in the UI and specify a value for **Application logo**.</li></ul>This value is used as the logo on the application's tile for the dashboard when you:<ul><li>Create an app in the UI and specify a value for **Application logo**.</li></ul>               | URL                                                                                            | TRUE       | FALSE    | FALSE      |
```

Comment:
It feels a little odd to be referencing creating an app via the UI here. It also is making it sound a bit like if you created the app via the UI with a logo specified and then change `logo_uri` the new value would be used for the application dashboard, which is _not_ the case.
I think I would structure this by keeping `URL string that references a logo for the client. This value is used with the client consent dialog box during the client consent flow.` here in this table, and breaking out the rules/way it is populated when the client is created in the UI as a note below the table.